cli_tools/common/utils/daisy/daisy_utils.go

```diff
@@ -84,28 +85,44 @@ func UpdateAllInstanceNoExternalIP(workflow *daisy.Workflow, noExternalIP bool)
 	if !noExternalIP {
 		return
 	}
-	updateAllInstanceAccessConfig(workflow, func() []*compute.AccessConfig {
-		return []*compute.AccessConfig{}
-	})
-}
-
-func updateAllInstanceAccessConfig(workflow *daisy.Workflow, accessConfigProvider func() []*compute.AccessConfig) {
-	for _, step := range workflow.Steps {
-		if step.IncludeWorkflow != nil {
-			//recurse into included workflow
-			updateAllInstanceAccessConfig(step.IncludeWorkflow.Workflow, accessConfigProvider)
-		}
+	WorkflowStepIterator(workflow, func(step *daisy.Step) {
 		if step.CreateInstances != nil {
 			for _, instance := range *step.CreateInstances {
 				if instance.Instance.NetworkInterfaces == nil {
 					return
 				}
 				for _, networkInterface := range instance.Instance.NetworkInterfaces {
-					networkInterface.AccessConfigs = accessConfigProvider()
+					networkInterface.AccessConfigs = []*compute.AccessConfig{}
 				}
 			}
 		}
-	}
+	})
+}
+
+// UpdateToUEFICompatible updates workflow resources (disks and images) to be UEFI compatible
+func UpdateToUEFICompatible(workflow *daisy.Workflow) {
+	WorkflowStepIterator(workflow, func(step *daisy.Step) {
+		if step.CreateDisks != nil {
```

Comment:
My gut feeling is that "daisy_utils.go" isn't the best place for this, given how specific it is to a particular use case.

Reply:
This is daisy_utils in cli_tools: Daisy utilities for CLI tools. This is used by both import wrapper and OVF importer. I think this is the most appropriate location for this piece of code. If you think there's a better place for it, please do suggest.

Reply:
A few places that come to mind:
 - Its own file
 - A file related to guest os features
 - A method on the workflow itself

Reply:
Those are all valid options. I feel the current location is good enough, considering the temporary nature of this feature. If you feel strongly against it, I will look into moving it

Reply:
Good point about it being temporary. Sounds like you're happy with it, so I'm fine!